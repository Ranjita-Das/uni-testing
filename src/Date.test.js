import React from 'react';
import DateComponent from './Date.js';
import renderer from 'react-test-renderer';

it('renders date correctly',()=>{
  const tree=renderer.create(<Date />).toJSON();
  expect(tree).toMatchSnapshot();
});
