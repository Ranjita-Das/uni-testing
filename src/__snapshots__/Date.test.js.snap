// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders date correctly 1`] = `"Fri Feb 16 2018 23:15:09 GMT-0800 (PST)"`;
